src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -258,6 +258,35 @@ public static boolean isAnyEmpty(final CharSequence... css) {
     }
     
     /**
+     * <p>Checks if any one of the CharSequences are not empty ("") or null.</p>
+     *
+     * <pre>
+     * StringUtils.isAnyNotEmpty(null)             = false
+     * StringUtils.isAnyNotEmpty(null, "foo")      = true
+     * StringUtils.isAnyNotEmpty("", "bar")        = true
+     * StringUtils.isAnyNotEmpty("bob", "")        = true
+     * StringUtils.isAnyNotEmpty("  bob  ", null)  = true
+     * StringUtils.isAnyNotEmpty(" ", "bar")       = true
+     * StringUtils.isAnyNotEmpty("foo", "bar")     = true
+     * </pre>
+     *
+     * @param css  the CharSequences to check, may be null or empty
+     * @return {@code true} if any of the CharSequences are empty or null
+     * @since 3.5
+     */
+    public static boolean isAnyNotEmpty(final CharSequence... css) {
+      if (ArrayUtils.isEmpty(css)) {
+        return true;
```

Comment:
Shouldn't this return false? An empty Array does not contain any non empty CharSequences. WDYT?